tools/cloud-build/provision/list_tests.py

```diff
@@ -43,6 +43,8 @@ def list_builds(start_time=30, end_time=5*60):
     interval = (end_time - start_time) // max(1, len(builds) - 1)
     res = {}
     for b in builds:
+        if b == "ofe-deployment":  # Skip ofe-deployment test.
```

Comment:
Add comment pointing future reader to the `tools/cloud-build/provision/pr-ofe-test.tf` where ofe is specially configured.

Reply:
Done.